server/index.js

```diff
@@ -1,29 +1,22 @@
 /* eslint global-require: 0 */
+import Hull from "hull";
+import express from "express";
+
+const { PORT = 8082, SECRET = "123", LOG_LEVEL, NODE_ENV = "development" } = process.env;
+
+let onMetric = function onMetric(metric, value, ctx) {
+  console.log(`[${ctx.id}] segment.${metric}`, value);
+};
 
 if (process.env.NEW_RELIC_LICENSE_KEY) {
   console.warn("Starting newrelic agent with key: ", process.env.NEW_RELIC_LICENSE_KEY);
   require("newrelic");
```

Comment:
we should remove the newrelic instrumentation here - it's done automatically by the `Hull.Connector()`.